### PR TITLE
GUACAMOLE-1064: add Norwegian keyboard

### PR DESCRIPTION
**Pull request by oharboe**
@jmoehlma Best guess... How do I test this?

### CODE REVIEW

**Comment by necouchman** [comment]
@oharboe: Looks fine without having an actual keyboard to test it with. Unfortunately I don't have a way to test, either :smile:

**Comment by oharboe** [comment]
@necouchman Is there a way to test this beyond manual integration testing, i.e. set up a couple of machines w/Guacamole?

**Comment by mike-jumper** [comment]
@oharboe, no, there is no automated test mechanism for keyboard layout behavior. The only way to test that a new keyboard layout behaves correctly is to deploy and directly test against an RDP connection using that layout.

**Comment by oharboe** [comment]
@mike-jumper Tested. Seems to work :-)

`sudo showkey` was useful to find scancodes.

File: src/protocols/rdp/Makefile.am [hunk 1]
```diff
@@ -241,7 +241,8 @@ rdp_keymaps =                                \
     $(srcdir)/keymaps/pt_br_qwerty.keymap    \
     $(srcdir)/keymaps/sv_se_qwerty.keymap    \
     $(srcdir)/keymaps/da_dk_qwerty.keymap    \
-    $(srcdir)/keymaps/tr_tr_qwerty.keymap
+    $(srcdir)/keymaps/tr_tr_qwerty.keymap    \
+    $(srcdir)/keymaps/no_no_qwerty.keymap
```
Comment (mike-jumper) [inline src/protocols/rdp/Makefile.am:245]: With the exception of "base" and "failsafe" (which are special cases) and "da_dk_qwerty" (which somehow slipped in here unsorted), the files listed here are being maintained in sorted order. Please move your new keymap up in the list such that sorted order is maintained.

**Review by mike-jumper — CHANGES_REQUESTED** [review]
Except for the ordering of the keymaps (see comment), this LGTM.

**Review by mike-jumper — APPROVED** [review]

**Pull request merged**